components/raftstore/src/store/fsm/peer.rs

```diff
@@ -444,9 +448,16 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
                 }
                 PeerMsg::Noop => {}
             }
+            RAFT_EVENT_DURATION
+                .with_label_values(&[typ])
```

Comment:
please use static metrics

Reply:
Do you mean the macro `make_auto_flush_static_metric!`? It's not able to use it in release-4.0...

Reply:
`make_static_metrics` should be enough